### PR TITLE
Update Issue templates

### PR DESCRIPTION
**Pull request by hexadecimal233**
See #86

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -8,15 +8,26 @@ body:
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this bug!
-        Make sure you are running the latest version before reporting.
-  - type: input
+        Thank you for taking your time to report this bug! You gotta make sure these before reporting...
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/bug_report.yml:11]: Wording can be improved here
```suggestion
        Thank you for taking your time to report this bug! Before opening an issue, please make sure that:
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -8,15 +8,26 @@ body:
...
+        - You are running the latest version of Connector and its dependencies.
+        - This bug wasn't already reported or being closed.
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/bug_report.yml:13]: Wording can be improved here
```suggestion
        - This bug wasn't already reported or closed as not planned.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -37,7 +48,7 @@ body:
       label: Logs
       description: |
         If applicable (crash, error output in console), please provide your crash report or latest.log
-        To upload logs, use an external site to post the crash report, such as [Ubuntu Pastebin](https://paste.ubuntu.com/), [Pastebin](http://pastebin.com/), or [Github Gist](https://gist.github.com/)
+        To upload logs, use an external site to post the crash report, such as [**MCLogs (Recommended)**](https://mclo.gs/), [Ubuntu Pastebin](https://paste.ubuntu.com/), or [Pastebin](http://pastebin.com/)
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/bug_report.yml:51]: Looks like paste.ubuntu.com requires registration now, and I could not find any information about its size limit. We should replace it with GitHub Gist instead, which still requires sign up but allows up to 100 MBs of data.
```suggestion
        To upload logs, use an external site to post the crash report, such as [**MCLogs (Recommended)**](https://mclo.gs/), [Github Gist](https://gist.github.com/), or [Pastebin](http://pastebin.com/)
```

File: .github/ISSUE_TEMPLATE/mod_malfunctioning.yml
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/mod_malfunctioning.yml:1]: Is there any reason to have separate mod malfunction and mod crash issue templates? Both cases are pretty similar, so I think we can keep using a unified mod compat template, and then label the issue once diagnosed accordingly.

File: .github/ISSUE_TEMPLATE/mod_crash.yml [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
 body:
   - type: markdown
     attributes:
       value: "## Welcome!"
   - type: markdown
     attributes:
       value: |
-        Thank you for taking your time to report this incompatibility!
-        Make sure you are running the latest version of Connector and its dependencies before reporting.
+        Thank you for taking your time to report this incompatibility! You gotta make sure these before reporting...
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/mod_crash.yml:11]: Wording can be imporved here
```suggestion
        Thank you for taking your time to report this incompatibility! Before opening an issue, please make sure that:
```

File: .github/ISSUE_TEMPLATE/mod_crash.yml [hunk 5]
```diff
@@ -1,21 +1,22 @@
...
+        - You are running the latest version of Connector and its dependencies.
+        - This bug wasn't already reported or being closed.
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/mod_crash.yml:13]: Wording can be improved here
```suggestion
        - This bug wasn't already reported or closed as not planned.
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+description: Feel free to give us suggestions and enhancement ideas!
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: "## Welcome!"
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking your time to give us suggestions! You gotta make sure these before reporting...
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/suggestion.yml:11]: Wording can be improved here
```suggestion
        Thank you for taking your time to give us suggestions! Before opening an issue, please make sure that:
```

File: .github/ISSUE_TEMPLATE/suggestion.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        - This feature does not exist in the latest build.
+        - This suggestion wasn't already reported or being closed.
```
Comment (Su5eD) [inline .github/ISSUE_TEMPLATE/suggestion.yml:13]: Wording can be improved here
```suggestion
        - This suggestion wasn't already reported or closed as not planned.
```

**Review by Su5eD — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/mod_malfunctioning.yml
Comment (Fiz-Victor) [reply .github/ISSUE_TEMPLATE/mod_malfunctioning.yml:1]: Yep, you've separated mod malfunction and crash in the compat thread, and crash should have a higher fix prio compared to malfunction.

**Comment by LLKawi** [comment]
There has been no updates for months and we do see the increasingly comment number in mod compat thread.
I also made this PR cuz mclo.gs provides a better log viewing experience and I've been wondering why it has not been merged.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (niklaswimmer) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: How about we just remove paste.ubuntu.com, see no reason why we would need it tbh (same for gist). pastebin.com seems fine as an alternative, but can we use a https link instead?

File: .github/ISSUE_TEMPLATE/mod_malfunctioning.yml
Comment (niklaswimmer) [reply .github/ISSUE_TEMPLATE/mod_malfunctioning.yml:1]: I agree with @Su5eD that those labels could be added by triage instead. I also noticed that the template now explicitly mentions modpacks - I think this is fine, but we should add a disclaimer of some sort that the user can use bisecting to identify which mod specifically causes problems. This process can take quite some time which makes plain "Modpack x is not working" issues not very actionable.

**Comment by SaintSelenya** [comment]
I never meant to actually post this, or have it be available at all 

i never used github before so apologies about all that

**Comment by hexadecimal233** [comment]
wip

**Pull request closed without merging**